src/layman/rest_multipublication_test.py

```diff
@@ -569,6 +583,21 @@ def test_get_publications(publication_type, headers, query_params, expected_publ
         (workspace2, authn_headers_user2, {'order_by': 'title',
                                            'full_text_filter': 'prilis',
                                            'bbox_filter': ','.join(str(c) for c in (2000, 2000, 6000, 6000)),
+                                           'bbox_filter_crs': crs_def.EPSG_3857,
+                                           'offset': 1,
+                                           'limit': 1,
+                                           }, [
+            # (workspace2, publication_2o_2_2x4_4),
+            (workspace2, publication_2e_3_3x5_5),
+        ], {'X-Total-Count': '2',
+            'Content-Range': 'items 2-2/2',
+            },
+        ),
+        (workspace2, authn_headers_user2, {'order_by': 'title',
+                                           'full_text_filter': 'prilis',
+                                           'bbox_filter': ','.join(str(c) for c in (0.0179663, 0.0179663, 0.0538989, 0.0538989)),
+                                           # EPSG:3857 (2000, 2000, 6000, 6000)
+                                           'bbox_filter_crs': crs_def.EPSG_4326,
```

Comment:
I've just realized that it is useful to test also bbox with negative numbers in REST API, i.e. EPSG:5514, to be sure that parsing of negative numbers works.